core/capability-negotiation-3.2.md

```diff
@@ -42,7 +42,8 @@ If no other arguments are present, version 3.1 is assumed.
 ### Capabilities with values
 
 Servers MAY specify additional data for each capability using the
-`<name>[=<value>]` format.
+`<name>[=<value>]` format for `CAP LS` and `CAP NEW`.  `CAP ACK` and
+`CAP NAK` replies MUST NOT contain any values.
```

Comment:
At the moment, I do not know of any servers that do this.  My concern was more for client-side parsing - if any servers offer a `CAP ACK` payload regardless of whether or not it differs, values would need to be discarded, e.g. ```if reply.contains('=') { reply.split('=')[0] […] }```.

That said, given the low likelihood of this being an issue (the examples encourage implementing the right way), it's probably less overhead to reach out to the authors of any IRC server who happen to have implemented it incorrectly.

I'll update the commit to not mention `CAP ACK` or `CAP NAK` at all.